app/js/fm-player/controllers/PlayerCtrl.js

```diff
@@ -80,15 +80,22 @@ angular.module("sn.fm.player").controller("PlayerCtrl", [
         };
 
         /**
-         * Increment volume by 10
+         * Value to increment/decrement volume by
+         * @property volumeStep
+         * @type     {Number}
+         */
+        $scope.volumeStep = 5;
+
+        /**
+         * Increment volume
          * @method volumeUp
          */
         $scope.volumeUp = function volumeUp() {
             PlayerVolumeResource.get({}, function(volume){
 
-                // round volume to nearest 10
-                volume.volume = Math.round(volume.volume / 10) * 10;
-                volume.volume = volume.volume + 10;
+                // round volume
+                volume.volume = Math.round(volume.volume / $scope.volumeStep) * $scope.volumeStep;
+                volume.volume = volume.volume + $scope.volumeStep;
 
```

Comment:
okay i see what your doing, one of those cases of maybe commenting your code in the future to make it clear